packages/api-client/src/odsql/clauses.ts

```diff
@@ -1,11 +1,36 @@
 import { field, string } from '.';
 
+const formatFields = (fields: string[]) => {
+    if (!fields || fields.length === 0) return '';
+    return `${fields.map(field).join(', ')},`;
+};
+
+/**
+ * Perform full-text search on all fields. It matches the text fields that contain strings beginning with the searched string.
+ * @param text_fields Fields to search in
+ * @param text Text to search for
```

Comment:
```suggestion
 * @param text Text to search for
 * @param text_fields Fields to search in
```

To follow args order

Same for `textStartWith` and `textSuggest`